CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ://github.com/handsontable/handsontable/pull/7162)
 - Fixed an issue where the column headers were cut off after hiding and then showing columns using the Hidden Columns
  plugin. [#6395](https://github.com/handsontable/handsontable/issues/6395)
+- Fixed an issue where the rows were missing their left border after disabling the row headers using `updateSettings
+`, while there were `fixedColumnsLeft` defined. [#5735](https://github.com/handsontable/handsontable/issues/5735)
```

Comment:
Please change to a single line.

Reply:
Is this an official rule? @wojciechczerniak

Reply:
No, but sounds good to establish one. I've noticed that line breaks broke the links and formatting (code block had additional space). And might be formatted differently depending on container max width (when we move changelog to release notes on github or to a blog post), and on mobile devices when the github is not displayed at the default width.

Reply:
> And might be formatted differently depending on container max width

I don't think a single line break creates a visual line break in markdown. But it's true, the auto hard break in my IDE broke the url to one of the entries, so maybe let's stick to not breaking lines at all.